.github/workflows/python-app.yml

```diff
@@ -63,8 +54,44 @@ jobs:
       - name: Build distribution package
         run: |
           python setup.py bdist_wheel
-      # - name: Publish distribution to PyPI
-      #   if: startsWith(github.ref, 'refs/tags')
-      #   uses: pypa/gh-action-pypi-publish@master
-      #   with:
-      #     password: ${{ secrets.PYPI_API_TOKEN }}
+      - name: Upload Python Wheel
+        uses: actions/upload-artifact@v2
+        with:
+          name: aac_wheel
+          path: dist/
+
+  test:
```

Comment:
Could we call this smoke test, functional test, acceptance test, or something to make it more explicit what's going on. If we just call it test, then I'd think we should move (at least) the unit test stage to this job.

Reply:
Good point, it is nondescript

Reply:
```suggestion
  Wheel Package Tests:
```